frontend/src/js/components/tenants/tenantlist.tsx

```diff
@@ -126,11 +131,32 @@ const TenantListItem = (props: ListItemComponentProps<Tenant>) => {
 };
 export const TenantList = () => {
   const tenantListState = useSelector(getTenantsList);
-  const { tenants, perPage, selectedTenant } = tenantListState;
+  const { tenants, perPage, selectedTenant, sort = {} } = tenantListState;
   const dispatch: AppDispatch = useDispatch();
   useEffect(() => {
     dispatch(getTenants());
   }, [dispatch]);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, setLocationParams] = useLocationParams('tenants', {
```

Comment:
the selected tenant information from the URL should probably be used here to set the `selectedTenant` in an effect